src/angularjs/src/app/components/analysis-jobs/analysis-jobs-status.filter.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @ngdoc filter
+ * @name pfb.analysis-jobs.status:displayStatus
+ *
+ * @description
+ * Transforms analysis job status into user friendly string
+ */
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function displayStatus() {
+        var statuses = {
+            CREATED: 'Created',
```

Comment:
Yeah, from the API is probably best, but we don't have an endpoint for that yet. Moving to a constants file and making each item in a dict follow the form:
```
var status = {
  IMPORTING: {
    short: 'shortdesc',
    long: 'longer description'
  },
  ...
}
```
Might be a good middle ground for now. In the place where keys/values are reversed, just loop the new status dict to find the key you want, it's not like that's going to be a big performance hit. 

Or just leave as is and make a separate issue to add the endpoint and fix this up, but either way #179 added an additional `QUEUED` status that should be reflected here before merging.